server/etcdserver/txn/txn.go

```diff
@@ -200,10 +198,12 @@ func Range(ctx context.Context, lg *zap.Logger, kv mvcc.KV, txnRead mvcc.TxnRead
 		}
 	}
 
+	resp.More = rr.More
 	if r.Limit > 0 && len(rr.KVs) > int(r.Limit) {
 		rr.KVs = rr.KVs[:r.Limit]
 		resp.More = true
 	}
```

Comment:
I intentionally didn't implement this for maxBytes. It seems not a big change. Let me see how to it.

Reply:
Actually I think we can even the following code, because `limit = limit + 1` was removed (line 136), and `len(rr.KVs) > int(r.Limit)` will never be true. But it isn't a big deal, so I did not raise the comment. 

```
if r.Limit > 0 && len(rr.KVs) > int(r.Limit) {
		rr.KVs = rr.KVs[:r.Limit]
		resp.More = true
	}
```

Reply:
Don't agree, please read 
https://github.com/etcd-io/etcd/blob/ddd65e7fe2e2d6fbb9a89a07ddbef79412885659/server/etcdserver/txn/txn.go#L127-L133

Condition `len(rr.KVs) > int(r.Limit)` can definitely be broken when sorting or filtering is enabled.

Reply:
Done